subsys/bluetooth/controller/Kconfig

```diff
@@ -20,6 +20,38 @@ config BLECTLR_SIGNAL_STACK_SIZE
 	  Size of the signal handler thread stack, used to process lower
 	  priority signals in the controller.
 
+comment "Multithreading behavior of BLE controller APIs"
+
+config BLECTLR_SHARED_API_BLOCKING
```

Comment:
I think it can be `menuconfig` instead of `config`. Then all `_API_BLOCKING` options could be hidden inside separate page in menuconfig